make warning from gcc

```
lzcomp.c: In function ‘optimize’:
lzcomp.c:312:12: warning: this statement may fall through [-Wimplicit-fallthrough=]
         if (commands -> value != next -> value) break;
            ^
lzcomp.c:313:7: note: here
       case 3:
       ^~~~
```

--- a/tools/lzcomp.c
+++ b/tools/lzcomp.c
@@ -310,6 +310,7 @@ void optimize (struct command * commands, unsigned short count) {
         break;
       case 1:
         if (commands -> value != next -> value) break;
+        // falls through
       case 3:
         if ((commands -> count + next -> count) <= 1024) {
           commands -> count += next -> count;

